scripts/common.js

```diff
@@ -0,0 +1,31 @@
+const { execSync } = require("child_process");
+
+module.exports = {
+    run: function run(command) {
+        return execSync(command).toString().trim()
+    },
+    getCommitId: function getCommitId() {
+        return this.run('git rev-parse --short HEAD');
+    },
+    determineVersion: function determineVersion() {
+        // Form version string - branch name and commit id
+        let commitId = this.getCommitId();
+        let branchName;
+        if (process.env.BRANCH_NAME) {
+            console.log('Using BRANCH_NAME from the environment')
+            branchName = process.env.BRANCH_NAME
+        } else {
+            console.log('Using local Git repo to determine branch name (environment BRANCH_NAME may also be used)')
+            branchName = this.run('git branch --show-current').replace(/\//g,"-")
```

Comment:
this seems like a new feature of git that my version doesn't have. Can you use `git rev-parse --abbrev-ref HEAD` instead for wider compatibility?

Reply:
Of course.